config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -165,20 +218,20 @@ periodics:
           cpu: 4
 
 # manual-release-bump-required
-- interval: 6h
+- interval: 2h
```

Comment:
unsure why the lower interval for the stable2 job

Reply:
:man_shrugging:

Reply:
done